app/templates/gentelella/users/events/index.html

```diff
@@ -90,5 +90,44 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
+    <script type="text/javascript">
```

Comment:
earlier also Iadded changes here only, for the adjustment of the columns .  that datatable thing if you remember , changes were done here .